install.sh

```diff
@@ -1,8 +1,88 @@
 #!/bin/bash
 
+# install the required modules.
 apt-get clean && apt-get update && apt-get install -y python-pip python-dev
 
 pip install influxdb
 pip install pyserial
 pip install spidev
 pip install py-opc
+
+# Let user enter the parameter to customize(location, frequency etc.)
```

Comment:
The documentation needs to be updated accordingly.

Instead of appending it to the end of the `install.sh` script, can you put all of this in a `setup.sh` file and document it.